### DISCUSSION TITLE
How to make screen move

### DISCUSSION THREAD

**Discussion by Maxi2022gt** [XeroOl/notitg-mirin#79] [General]
how do i make the screen move? (not the window, the game screen)

**Comment by XeroOl** [comment]
There's sort of two ways to make this effect:

1) You can get the screen with `SCREENMAN:GetTopScreen()`, and then move that like any other actor.
2) Or, if you just want to move everything, without moving the actual screen, You can put a `Name=` on the outer ActorFrame of the `lua/layout.xml` to store it into a variable, and then move it like any other actor.
